libp2p/Common.cpp

```diff
@@ -253,5 +253,22 @@ boost::log::formatting_ostream& operator<<(boost::log::formatting_ostream& _log,
     return _log << _node.id << '@' << _node.endpoint;
 }
 
+std::unordered_map<Public, std::string> defaultBootNodes()
```

Comment:
I think it would be better to put it into local static variable and return const  reference to avoid extra copies

Reply:
Good idea, will do that